composer.json

```diff
@@ -15,7 +15,8 @@
         "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "yoast/phpunit-polyfills": "^1.0.0"
+        "yoast/phpunit-polyfills": "^1.0.0",
+        "phpunit/phpunit": "^9.5"
```

Comment:
You don't need to add this. yoast/phpunit-polyfills will add phpunit according to the constraints.